Sources/ProjectSpec/TargetScheme.swift

```diff
@@ -33,7 +35,8 @@ public struct TargetScheme: Equatable {
         commandLineArguments: [String: Bool] = [:],
         environmentVariables: [XCScheme.EnvironmentVariable] = [],
         preActions: [Scheme.ExecutionAction] = [],
-        postActions: [Scheme.ExecutionAction] = []
+        postActions: [Scheme.ExecutionAction] = [],
+        wasCreatedForAppExtension: Bool = false
```

Comment:
Let's use the default value here from above `wasCreatedForAppExtensionDefault`

Reply:
okay.